components/org.wso2.carbon.identity.oauth.par/src/main/java/org/wso2/carbon/identity/oauth/par/common/SQLQueries.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
```

Comment:
The license header is inconsistent. some files have `all rights reserved` and some don't. Check whats the header being used now and use that in all places